plugins/tiny.go

```diff
@@ -0,0 +1,54 @@
+package plugins
+
+import (
+	"net/http"
+	"bytes"
+	"encoding/json"
+	"github.com/belak/irc"
+	"github.com/belak/seabird/bot"
+	"github.com/belak/seabird/mux"
+)
+
+type ShortenResult struct {
+	Kind string `json:"kind"`
+	Id string `json:"id"`
+	LongUrl string `json:"longUrl"`
+}
+
+func init() {
+	bot.RegisterPlugin("tiny", NewTinyPlugin)
+}
+
+func NewTinyPlugin(m *mux.CommandMux) error {
+	m.Event("tiny", Shorten)
+
+	return nil
+}
+
+func Shorten(c *irc.Client, e *irc.Event) {
+	if e.Trailing() == "" {
+		c.MentionReply(e, "URL required")
+		return
+	}
+
+	url := "https://www.googleapis.com/urlshortener/v1/url"
+
+	var jsonStr = []byte(`{"longUrl":"` + e.Trailing() + `"}`)
```

Comment:
This should probably be escaped. I know _someone_ will try it. Little Bobby Tables...